hack/make/dynbinary-proxy

```diff
@@ -8,6 +8,7 @@ set -e
 	export BUILDFLAGS=("${BUILDFLAGS[@]/osusergo /}")     # ditto for osusergo
 	export BUILDFLAGS=("${BUILDFLAGS[@]/static_build /}") # we're not building a "static" binary here
 
+	DOCKER_STATIC=0
```

Comment:
Guess this is redundant, or do we explicitly check for `if DOCKER_STATIC = 0` anywhere?